### DISCUSSION TITLE
More assignable side button actions.

### DISCUSSION THREAD

**Discussion by firewuff** [egzumer/uv-k5-firmware-custom#548] [Ideas]
Firstly, this is amazing work and thankyou for all the effort.  This firmware is so much more user friendly than many others out there.  One function that would be awesome and make the UV series radios more accessible to a lot of users is a greater list of assignable actions to the FN1 and FN2 buttons.  the IJV mod has a long list of side key actions (https://www.universirius.com/en_gb/preppers/quansheng-uv-k5-manuale-del-firmware-ijv_v3/#Tasti, section 5.4) but the most useful to make this radio easy to use in dual RX mode is VFO Change and VFO Swap.  ideally just being able to assign FN1 to VFO1 and FN2 to VFO2 would make it easy to set up the radio for use by inexperienced users.  Thanks for the consideration.

**Comment by salokinrekced** [comment]
Yes! Please add the ability to map the Spectrum to an FN button and return the NOAA function to F+5.
